lib/core/stores/remote.ts

```diff
@@ -266,6 +267,116 @@ export class RemotePromiseStore implements IPromiseStore {
     }
   }
 
+  async createSchedule(
+    id: string,
+    ikey: string | undefined,
+    description: string | undefined,
+    cron: string,
+    tags: Record<string, string> | undefined,
+    promiseId: string,
+    promiseTimeout: number,
+    promiseHeaders: Record<string, string> | undefined,
+    promiseData: string | undefined,
+    promiseTags: Record<string, string> | undefined,
+  ): Promise<Schedule> {
+    const reqHeaders: Record<string, string> = {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    };
+
+    if (ikey !== undefined) {
+      reqHeaders["Idempotency-Key"] = ikey;
+    }
+
+    const schedule = this.call(`${this.url}/schedules`, this.isSchedule, {
+      method: "POST",
+      headers: reqHeaders,
+      body: JSON.stringify({
+        id,
+        description,
+        cron,
+        tags,
+        promiseId,
+        promiseTimeout,
+        promiseHeaders,
+        promiseData: promiseData ? this.encode(promiseData) : undefined,
+        promiseTags,
+      }),
+    });
+
+    return schedule;
+  }
+
+  async getSchedule(id: string): Promise<Schedule> {
+    const schedule = this.call(`${this.url}/schedules/${id}`, this.isSchedule, {
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+      method: "GET",
+    });
+
+    return schedule;
+  }
+
+  async deleteSchedule(id: string): Promise<boolean> {
+    try {
+      await this.call(`${this.url}/schedules/${id}`, this.isDeletedResponse, {
```

Comment:
We don't use the body returned from this call so I believe we can just to `(b: unknown): b is unknown => true` as a guard here